### ISSUE TITLE
reclassify tool: provide an area of interest

### ISSUE THREAD

**Issue by dfguerrerom** [12rambau/sepal_ui#250]
In the previous version (I'm afraid that I didn't push those changes) there was the option to include an AOI model and clip the reclassification inputs to that area before doing the calculation steps, and restrict everything to that area (if an AOI was selected), what do you think about reimplementing it?

**Comment by 12rambau** [comment]
yep that wasn't there when I refactored the file. Seems like an excellent idea to me.
